bundle/Event/InformationCollected.php

```diff
@@ -67,10 +75,52 @@ public function getLocation()
      * Returns additional content
      * This can be ez content or site api content.
      *
-     * @return Content
+     * @return Content|null
      */
     public function getAdditionalContent()
     {
         return $this->additionalContent;
     }
+
+    /**
+     * @return array
+     */
+    public function getAdditionalParameters()
+    {
+        return $this->additionalParameters;
+    }
+
+    /**
+     * @param array $additionalParameters
+     */
+    public function setAdditionalParameters(array $additionalParameters)
+    {
+        $this->additionalParameters = $additionalParameters;
+    }
+
+    /**
+     * @param string $key
+     * @param mixed $value
+     * @return self
+     */
+    public function setAdditionalParameter(string $key, $value)
```

Comment:
You can't add scalar typehints since 1.x of this bundle still supports PHP 5.6

Reply:
Changed in https://github.com/netgen/NetgenInformationCollectionBundle/pull/76/commits/05a2bd4537bca7079bcdeca78a2112af992ae551